pdns/rpzloader.cc

```diff
@@ -205,6 +207,10 @@ shared_ptr<SOARecordContent> loadRPZFromServer(const ComboAddress& master, const
       RPZRecordToPolicy(dr, zone, true, defpol, maxTTL);
       nrecords++;
     } 
+    axfrNow = time(nullptr);
+    if (axfrNow - axfrStart > axfrTimeout) {
```

Comment:
Perhaps we should check that `axfrNow` >= `axfrStart` in case the time goes backward because of NTP adjustment for example